testfiles/5ZoneAirCooledConvCoef_VSFan.idf

```diff
@@ -0,0 +1,3591 @@
+    0.0,                     !- Electric Power Minimum Flow Rate Fraction
+    50.0,                    !- Design Pressure Rise {Pa}
+    0.9,                     !- Motor Efficiency
+    1.0,                     !- Motor In Air Stream Fraction
+    AUTOSIZE,                !- Design Electric Power Consumption {W}
+    TotalEfficiencyAndPressure,  !- Design Power Sizing Method
+    ,                        !- Electric Power Per Unit Flow Rate {W/(m3/s)}
+    ,                        !- Electric Power Per Unit Flow Rate Per Unit Pressure {W/((m3/s)-Pa)}
+    0.50,                    !- Fan Total Efficiency
+    CombinedPowerAndFanEff;  !- Electric Power Function of Flow Fraction Curve Name
+
+  AirLoopHVAC:ZoneMixer,
+    SPACE2-1 PIU Mixer,      !- Name
+    SPACE2-1 Zone Coil Air In Node,  !- Outlet Node Name
+    SPACE2-1 ATU In Node,    !- Inlet 1 Node Name
+    SPACE2-1 ATU Fan Outlet Node;  !- Inlet 2 Node Name
+
+  Coil:Heating:Water,
+    SPACE2-1 Zone Coil,      !- Name
+    ReheatCoilAvailSched,    !- Availability Schedule Name
+    autosize,                !- U-Factor Times Area Value {W/K}
+    autosize,                !- Maximum Water Flow Rate {m3/s}
+    SPACE2-1 Zone Coil Water In Node,  !- Water Inlet Node Name
+    SPACE2-1 Zone Coil Water Out Node,  !- Water Outlet Node Name
+    SPACE2-1 Zone Coil Air In Node,  !- Air Inlet Node Name
+    SPACE2-1 In Node,        !- Air Outlet Node Name
+    UFactorTimesAreaAndDesignWaterFlowRate,  !- Performance Input Method
+    autosize,                !- Rated Capacity {W}
+    82.2,                    !- Rated Inlet Water Temperature {C}
+    16.6,                    !- Rated Inlet Air Temperature {C}
+    71.1,                    !- Rated Outlet Water Temperature {C}
+    32.2,                    !- Rated Outlet Air Temperature {C}
+    ;                        !- Rated Ratio for Air and Water Convection
+
+  AirTerminal:SingleDuct:SeriesPIU:Reheat,
+    SPACE3-1 VAV Reheat,     !- Name
+    ReheatCoilAvailSched,    !- Availability Schedule Name
+    autosize,                !- Maximum Air Flow Rate {m3/s}
+    autosize,                !- Maximum Primary Air Flow Rate {m3/s}
+    autosize,                !- Minimum Primary Air Flow Fraction
+    SPACE3-1 ATU In Node,    !- Supply Air Inlet Node Name
+    SPACE3-1 ATU Sec Node,   !- Secondary Air Inlet Node Name
+    SPACE3-1 In Node,        !- Outlet Node Name
+    SPACE3-1 Zone Coil Air In Node,  !- Reheat Coil Air Inlet Node Name
+    SPACE3-1 PIU Mixer,      !- Zone Mixer Name
+    SPACE3-1 PIU Fan,        !- Fan Name
+    Coil:Heating:Water,      !- Reheat Coil Object Type
+    SPACE3-1 Zone Coil,      !- Reheat Coil Name
+    autosize,                !- Maximum Hot Water or Steam Flow Rate {m3/s}
+    0.0,                     !- Minimum Hot Water or Steam Flow Rate {m3/s}
+    0.0001,                  !- Convergence Tolerance
+    VariableSpeed,
+    0.5,
+    Staged,
+    ,
+    ;
```

Comment:
I'm super late to the party, but this is unfortunate this doesn't have the field comments, as grepping for "Heating Control Type" in the testfiles/ folder returns nothing. I was trying to find an example file to test this is new feature.